app/crud/api_keys.py

```diff
@@ -0,0 +1,65 @@
+import uuid
+from datetime import datetime
+from typing import List, Optional
+
+from app.errors import RecordNotFoundError
+from app.models.orm.api_keys import ApiKey as ORMApiKey
+
+
+async def create_api_key(
+    user_id: str,
+    alias: Optional[str],
+    organization: str,
+    email: str,
+    domains: List[str],
+) -> ORMApiKey:
+
+    # If a simple string is used for domains, sqlalchemy will still inject this into db,
+    # using every character as a list item. We don't want this to happen
+    assert isinstance(domains, list), "Domains must be of type List[str]"
```

Comment:
shouldn't this already be caught by the List[str] type for the parameter? or are there paths that a string will come instead of a list?

Reply:
I noticed while testing that when adding strings it will become funky. `List[str]` is only a type hint and not enforced. I prefer to catch this for sure before injecting it into the DB. You never know.